wbooks/App.js

```diff
@@ -16,19 +16,19 @@ import {
 } from 'react-native';
 
 import BookList from './src/app/screens/booklist';
-import styles from './styles';
+import {createAppContainer} from 'react-navigation';
```

Comment:
leave space between brackets

Reply:
This import should be with the `node_modules` imports.

Reply:
done